graylog2-web-interface/src/components/navigation/Navigation.styles.jsx

```diff
@@ -84,7 +84,8 @@ const StyledNavbar = styled(Navbar)(({ theme }) => css`
       }
 
       #scratchpad-toggle,
-      .dropdown-toggle {
+      .dropdown-toggle,
+      #welcome-nav-link {
```

Comment:
Do we really need this id? I can't see a difference, when I remove it.

Reply:
@maxiadlovskii can you please take a look?